javascript/javascript1/week1/pizza-exercise/part4.js

```diff
@@ -0,0 +1,32 @@
+console.log("==== PART 4====");
+
+const orderNumber2 = [1, 2, 2]; // These are Pesto, Pep og Pep without any discounts
+console.log("Order:", orderNumber2);
+
+const totalPrice2 =
+  pizzaPrices[order[1]] + pizzaPrices[order[2]] + pizzaPrices[order[2]];
+console.log("Total price:", totalPrice2.toFixed(2), "DKK");
+
+// Adding 20% off the first pizza
+const discount2 = pizzaPrices[order[0]] * 0.2;
```

Comment:
Up in `totalPrice2` your first pizza is `order[1]`.
```suggestion
const discount2 = pizzaPrices[order[1]] * 0.2;
```